### DISCUSSION TITLE
Include nested documents in templates

### DISCUSSION THREAD

**Discussion by gemathus** [outline/outline#3349] [Ideas]
The template feature is awesome. Nevertheless, I think that being able to include nested documents in a template could be really useful. 

As of now (v. 0.62.0), I can only create a template from a single document. If I have nested documents in the template's base document, it will not include them in the new template.

Is this something that can be done or do we have to develop the feature?

**Comment by tommoor** [comment]
The feature doesn't exist today but it's probably not too difficult to develop - after all, the primitives exist

**Reply by gemathus** [reply]
@tommoor Thanks! 👍

**Reply by antran22** [reply]
Bumping this functionality. @tommoor can you give us a few pointer on how one may approach this and I will try to make a quick PoC.
